### DISCUSSION TITLE
Verification of Student Developer Pack

### DISCUSSION THREAD

**Discussion by TheAetherLord** [community/community#56629] [GitHub Education]
### Hi everyone,

Hello, I've been trying to apply for the GitHub student developer pack for quite some time and it was giving me errors that my profile isn't complete. After doing everything to fix that I got a message as 'Thank you for submitting, we'll get back to you within one day' or something like that but when I didn't receive mail for a whole 3 days I checked again. It says they couldn't approve it due to my profile being incomplete so I tried again today and now I started to get a new type of error. (Refer to the screenshot)
![image](https://github.com/community/community/assets/70020209/b52444cb-c2c7-4004-bff5-bcb286aad380)
I was hoping if there's any manual way to get approval as I really am a student I can provide my student ID card along with university registrations and marksheets of previous semesters also the receipt from my admission please let me know what should I do?

**Comment by Ntshangase** [comment]
Hello @TheAetherLord 

- It's recommended that after you update your profile you sign-out of GitHub and then Sign-in again 
- make sure that the documents you are submitting are clear and dated.

**Reply by TheAetherLord** [reply]
well you can check my profile and can tell me if I'm missing anything I mean it all looks covered to me but still cant seem to get it.

**Comment by samus-aran** [comment]
Hi @TheAetherLord 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
